saleor/graphql/account/mutations/permission_group.py

```diff
@@ -300,6 +272,33 @@ def clean_users(
             )
             cls.clean_remove_users(requestor, errors, cleaned_input, group)
 
+    @classmethod
+    def ensure_can_manage_users(
+        cls,
+        requestor: "User",
+        errors: Dict[Optional[str], List[ValidationError]],
+        field: str,
+        cleaned_input: dict,
+    ):
+        """Check if requestor can manage users from input.
+
+        Requestor cannot manage users with wider scope of permissions.
+        """
+        users = cleaned_input[field]
```

Comment:
var used once but it's good to leave it for readability since i wouldn't know on 1st glance what is `cleaned_input[field]`. Though, could be moved below `if` as this only needs execution if `requestor` is not superuser.